### ISSUE TITLE
[BITV] 9.4.1.2/8.4 - The "Add group" button has been wrongly implemented as a link with the aria-expanded attribute.

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37135]
It should be implemented using a button element without the aria-expanded attribute. (1)

- [ ] Fix
- [ ] Backport

![4b98244143224a248f230b7ff75c24c8](https://user-images.githubusercontent.com/1315170/223751895-dfaef275-1184-4c7a-9245-2616d170fdd8.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v8-n4

</details>
